include/SANDDigitizationEDEPSIM.h

```diff
@@ -48,12 +48,34 @@ void group_hits_by_tube(TG4Event* ev, const SANDGeoManager& geo,
 
 void create_digits_from_hits(const SANDGeoManager& geo,
                              std::map<int, std::vector<hit> >& hits2Tube,
-                             std::vector<dg_tube>& digit_vec);
+                             std::vector<dg_wire>& digit_vec);
 
 void digitize_stt(TG4Event* ev, const SANDGeoManager& geo,
-                  std::vector<dg_tube>& digit_vec);
+                  std::vector<dg_wire>& digit_vec);
 }  // namespace stt
 
+namespace chamber
+{
+void group_hits_by_wire(TG4Event* ev, const SANDGeoManager& geo,
```

Comment:
Use `TG4Event&  ev`  or `TG4Event const&  ev` instead of `TG4Event* ev`
Passing an argument as a pointer makes sense if the object is expected to possibly be `nullptr `and the function includes a check for it. Otherwise,  it would be preferable to pass the object by reference.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f60-prefer-t-over-t-when-no-argument-is-a-valid-option

This comment applies to other parts of the code as well.